### DISCUSSION TITLE
Website Display Discrepancies between Chrome and Safari Mobile Browsers on cPanel Hosting

### DISCUSSION THREAD

**Discussion by shenaorj** [tailwindlabs/tailwindcss#12250] [Help] [answered]
Hello, I created a project for a campaign, and it looks perfect in the browser, and it's responsive. However, when I upload the project to cPanel, it doesn't display the same way in Chrome and Safari mobile browsers. The floating menu appears without text, and the videos only show the play button. Can you help me understand what's happening? Here are the links:
GitHub Repository: https://github.com/shenaorj/TecnologiaBiomedica.git
Live Site: https://www.tecnologia-biomedica.siingmi.com/

**Comment by alex-krasikau** [comment]
Hello @shenaorj, could you please send a screenshot showing the difference between what you have locally and the deployed version? This will help us identify the problematic area.

**Reply by shenaorj** [reply]
Hello @alex-krasikau sure!

1. Floating menu for mobile version,

- locally it looks like this:

![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/4fbf2122-9628-4323-8662-ec5768ca5e88)
- deployed version viewed in the browser. (It looks white, there are no letters.)
![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/5ff066bf-2496-41bb-bb3f-129d875dede8)

2. Video thumbnails,
- locally it looks like this:
![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/f2c7154f-ce48-4aef-874e-290e0f1208ed)
-deployed version viewed in the browser. (There's no video thumbnail, only the play button (which is functional).)
![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/7cb612f2-c849-4368-9aae-7e6ae2278fcd)

3. Button colors in the tips section,
-locally it looks like this:
![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/b700eb08-f0ab-4bd4-bbf9-f73d0de3cec5)
-deployed version viewed in the browser. (It's not the color)
![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/0eac0b33-6c3f-47fb-b1bc-b5b07ed56254)

4. Footer background color,
locally it looks like this:
- ![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/5667bca5-6352-46e6-acb7-c61c4f76e0f0)
- deployed version viewed in the browser. (It's not the color)
![image](https://github.com/tailwindlabs/tailwindcss/assets/112667326/88b4b677-0112-4762-81eb-3c3be135e172)


Annotation: In the desktop version, everything looks perfect on all the sites.

You're welcome!

**Reply by alex-krasikau** [reply, marked as answer]
On the computer, you see a light-themed version, but on mobile, you're seeing a dark-themed version. The website uses the theme defined by the browser or operating system. This means that on desktop, you haven't set the theme, but on mobile, you have dark mode turned on.

If you would like to test the appearances on your local machine, you can go to the operating system settings and toggle the appearance to dark mode. For a temporary test, you can open dev tools in Chrome, press "Cmd+Shift+P" (on Mac), enter "Emulate," and select the desired color scheme.
<img width="463" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/97882909/92a44713-1cb5-4abb-81c1-ea123870452b">

**Reply by shenaorj** [reply]
You're welcome, bro! Everything worked perfectly, and now you can review it. It was just a matter of removing the dark theme. Thank you very much!

By any chance, do you know the issue or problem with video thumbnails on mobile?

**Reply by alex-krasikau** [reply]
Not really, but most likely it's about the differences in the native implementation by different devices. For an easier implementation, I would recommend sticking to the existing hosting services that perform the user experience, compression, and other useful tasks for you. I guess, in 95% of cases, people use either Vimeo or YouTube.

**Reply by shenaorj** [reply]
Ah, I see. Yes, I think I will use YouTube.

Thank you very much for your support!
